src/App.tsx

```diff
@@ -1,24 +1,66 @@
+/* eslint-disable no-console */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
+import { TodoList } from './Components/TodoList/TodoList';
+import { FilterBy } from './utils/FilterBy';
+import { TodoFilter } from './Components/TodoFilter/TodoFilter';
+
+const USER_ID = 10341;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.All);
+
+  const visibleTodos = todos.filter((todo) => {
+    switch (filterBy) {
+      case FilterBy.Active:
+        return !todo.completed;
+
+      case FilterBy.Completed:
+        return todo.completed;
+
+      default:
+        return true;
+    }
+  });
+
+  const itemsLeft = todos.filter((todo) => !todo.completed).length;
+  const isTodosNoEmpty = todos.length > 0;
+
+  useEffect(() => {
+    client.get<Todo[]>(`/todos?userId=${USER_ID}`)
```

Comment:
make a separate function for it
```suggestion
    getTodos()
```